<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="tabletTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="moveBtn">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>105</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>移动当前界面</string>
    </property>
   </widget>
   <widget class="QPushButton" name="getWindowInfoBtn">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>120</y>
      <width>105</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>获取当前界面信息</string>
    </property>
   </widget>
   <widget class="QPushButton" name="modifyBtn">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>180</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>修改当前界面位置及尺寸</string>
    </property>
   </widget>
   <widget class="QPushButton" name="modifyIconOrTitleBtn">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>60</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>修改当前界面图标或标题</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
